docs/site/todo-list-tutorial-model.md

```diff
@@ -0,0 +1,115 @@
+---
+lang: en
+title: 'Add TodoList Model'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/todo-list-tutorial-model.html
+summary: LoopBack 4 TodoList Application Tutorial - Add TodoList Model
+---
+
+### Models with relations
+
+A set of data can often be related to another; an entity may be able to provide
+access to another based on its relationship with the other entity. To take
+`TodoListApplication` one step further and establish relations with the existing
+`Todo` model as real-world applications often tend to do, we'll introduce the
+model `TodoList`.
+
+### Building a checklist for your Todo models
+
+A todo item is often grouped into a checklist along with other todo items so
```

Comment:
I think this heading and paragraph might be a better start to this page ... and then jumping into talking about `models with relations` followed by the second paragraph of this section